ui/components/app/signature-request-siwe/signature-request-siwe-message/signature-request-siwe-message.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const SignatureRequestSIWEMessage = ({ data }) => {
+  return (
+    <div className="signature-request-siwe-message">
+      <div className="signature-request-siwe-message__list">
+        {data.map(({ label, value }, i) => (
+          <div className="signature-request-siwe-message__list__item" key={i}>
```

Comment:
We should ensure the key is a string value
ref: https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
Done